common/test/acceptance/tests/discussion/test_cohort_management.py

```diff
@@ -294,6 +304,177 @@ def test_add_new_cohort(self):
+                "time": {"$gt": start_time},
+                "event.cohort_name": cohort_name,
+            }).count(),
+            1
+        )
+
+        if verify_updated:
+            self.cohort_management_page.select_cohort(cohort_name)
+            self.cohort_management_page.select_cohort_settings()
+            self.cohort_management_page.set_cohort_name(new_cohort_name)
+            self.cohort_management_page.set_assignment_type(new_assignment_type)
+            self.cohort_management_page.save_cohort_settings()
+
+            # new cohort name should be present in the list of cohorts. But when we try to update an existing cohort
+            # with an empty cohort name then existing/old cohort name will be used because cohort name can't ba empty
+            new_cohort_name = new_cohort_name or cohort_name
+
+            confirmation_messages = self.cohort_management_page.get_cohort_settings_messages()
+            self.assertEqual(["Saved cohort"], confirmation_messages)
+            self.assertEqual(new_cohort_name, self.cohort_management_page.cohort_header_title)
+            self.assertTrue(new_cohort_name in self.cohort_management_page.get_cohorts())
+            self.assertEqual(1, self.cohort_management_page.get_selected_cohort_count())
+            self.assertEqual(new_assignment_type, self.cohort_management_page.get_cohort_associated_assignment_type())
+
+    def test_add_new_cohort_with_manual_assignment_type(self):
+        """
+        Scenario: A new manual cohort with manual assignment type can be created, and a student assigned to it.
+
+        Given I have a course with a user in the course
+        When I add a new manual cohort with manual assignment type to the course via the LMS instructor dashboard
+        Then the new cohort is displayed and has no users in it
+        And assignment type of displayed cohort is "manual"
+        And when I add the user to the new cohort
+        Then the cohort has 1 user
+        And appropriate events have been emitted
+        """
+        cohort_name = str(uuid.uuid4().get_hex()[0:20])
+        self._verify_cohort_settings(cohort_name=cohort_name, assignment_type='manual')
+
+    def test_add_new_cohort_with_random_assignment_type(self):
+        """
+        Scenario: A new manual cohort with random assignment type can be created, and a student assigned to it.
+
+        Given I have a course with a user in the course
+        When I add a new manual cohort with random assignment type to the course via the LMS instructor dashboard
+        Then the new cohort is displayed and has no users in it
+        And assignment type of displayed cohort is "random"
+        And when I add the user to the new cohort
+        Then the cohort has 1 user
+        And appropriate events have been emitted
+        """
+        cohort_name = str(uuid.uuid4().get_hex()[0:20])
+        self._verify_cohort_settings(cohort_name=cohort_name, assignment_type='random')
```

Comment:
Perhaps also verify that self.cohort_management_page.is_assignment_settings_disabled is False and self.cohort_management_page.assignment_settings_message is empty because there are 2 random cohorts.